docs/modules/deployment/pages/cloud-platforms.adoc

```diff
@@ -0,0 +1,327 @@
+
+.Create secrets for Admin API credentials
+[source,bash]
+----
+flyctl secrets set CERBOS_ADMIN_USER=cerbos <1>
+flyctl secrets set CERBOS_ADMIN_PASSWORD=$(echo "cerbosAdmin" | htpasswd -niBC 10 cerbos | cut -d ':' -f 2 | base64 -w0) <2>
+----
+<1> Specify username for Admin API
+<2> Specify password for Admin API
+
+.Create a Dockerfile
+[source,Dockerfile,subs="attributes+"]
+----
+FROM flyio/litefs:0.5 AS litefs
+
+FROM ghcr.io/cerbos/cerbos:{app-version} AS cerbos
+
+FROM alpine:3.16 AS base
+RUN apk add fuse3 sqlite
+ADD litefs.yml /etc/litefs.yml
+COPY --from=cerbos /cerbos /cerbos
+COPY --from=litefs /usr/local/bin/litefs /usr/local/bin/litefs
+
+ENTRYPOINT ["litefs"]
+CMD ["mount"]
+----
+
+.Create a `fly.toml` to launch Cerbos
+[source,toml,linenums,subs="attributes+"]
+----
+app = '<APPLICATION_NAME>' <1>
+primary_region = '<REGION>' <2>
+
+[build]
+  dockerfile = "Dockerfile"
+
+[mounts]
+  source = "litefs"
+  destination = "/var/lib/litefs" <3>
+
+[[services]]
+  protocol = ''
+  internal_port = 3592
+
+[[services.ports]]
+    port = 3592
+    handlers = ['tls', 'http']
+
+[[services.http_checks]]
+    interval = '5s'
+    timeout = '2s'
+    grace_period = '5s'
+    method = 'get'
+    path = '/_cerbos/health'
+    protocol = 'http'
+
+[[services]]
+  protocol = ''
+  internal_port = 3593
+
+[[services.ports]]
+    port = 3593
+    handlers = ['tls']
+
+    [services.ports.tls_options]
+      alpn = ['h2']
+
+[[vm]]
+  memory = '1gb'
+  cpu_kind = 'shared'
+  cpus = 1
+
+[metrics]
+  port = 3592
+  path = "/_cerbos/metrics"
+----
+<1> The name of the link:https://fly.io/docs/apps[Fly App]
+<2> Pick a link:https://fly.io/docs/reference/regions/#fly-io-regions[region]
+<3> Destination must be equal to the one specified in the `litefs.yaml`
+
+.Attach to Consul to manage LiteFS leases.
+[source,bash]
+----
+flyctl consul attach
+----
+
+NOTE: See link:https://fly.io/docs/litefs/getting-started-fly/#lease-configuration[lease configuration] for more
+information about Consul leases on Fly.io.
+
+After launching Cerbos with LiteFS, it is possible to add, disable, list or update policies by using
+link:https://docs.cerbos.dev/cerbos/latest/cli/cerbosctl.html#put[`cerbosctl`].
+
+.List policies with cerbosctl
+[source,bash,linenums]
+----
+cerbosctl \
+  --server=https://<APPLICATION_NAME>.fly.dev \ <1>
+  --username="cerbos" \
+  --password="cerbosAdmin" \
+  get derived_roles
+
+cerbosctl \
+  --server=https://<APPLICATION_NAME>.fly.dev \ <1>
+  --username="cerbos" \
+  --password="cerbosAdmin" \
+  get export_variables
+
+cerbosctl \
+  --server=https://<APPLICATION_NAME>.fly.dev \ <1>
+  --username="cerbos" \
+  --password="cerbosAdmin" \
+  get principal_policies
+
+cerbosctl \
+  --server=https://<APPLICATION_NAME>.fly.dev \ <1>
+  --username="cerbos" \
+  --password="cerbosAdmin" \
+  get resource_policies
```

Comment:
Just one example is enough because we have others in the cerbosctl documentation.

```suggestion

```